src/components/ImportExport.js

```diff
@@ -120,12 +119,13 @@ class ImportExport extends React.Component {
     const { trails, hydrants } = this.props;
     const { selectedExport } = this.state;
 
-    const trailFeatures = _.values(trails.toJS()).map(item => item.feature);
-    const hydrantFeatures = _.values(hydrants.toJS()).map(item => item.feature);
+    const trailFeatures = _.flatMap(_.values(trails.toJS()), item => item.features);
+    const hydrantFeatures = _.flatMap(_.values(hydrants.toJS()),item => item.features);
```

Comment:
Try out something like this instead:
https://github.com/jolavb/Sno_Demo/pull/49/files#diff-773001ae59bca40bfa36722e6ec50465R25

The `toJS` function is relatively expensive computationally I believe, the reduce should be a little more efficient.